packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/l10n/network_lookup/notifications_lookup.dart

```diff
@@ -22,9 +22,6 @@ class AppI18nNotificationsLookup extends I18nFeatureNotificationsLookup {
               I18nFeatureNotifications(I18nFeatureNotificationsLookup_bg());
           break;
         case 'en':
-          bundledI18n =
```

Comment:
Please make sure such changes are addressed in separate PRs. From what I see this is not related to QR code integration, but feel free to correct me if I'm wrong